docs/src/tutorials/conic/ellipse_approx.jl

```diff
@@ -109,7 +109,7 @@ m, n = size(S)
     [i in 1:m],
     S[i, :]' * Z * S[i, :] - 2 * S[i, :]' * z + s <= 1,
 )
-@constraint(model, [t; vec(Z)] in MOI.RootDetConeSquare(n))
```

Comment:
We use the syntax in examples, but what about documenting it in the general case, along with its limitations?
> Effectively, the tuple vectorized by concatenating the elements, and doesn't concern itself with whether the structure of the tuple makes sense for the set.

Reply:
I guess my concern was more: is this a good idea and should we move forward with it. It adds two ways to write each constraint (vector or tuple) which might be confusing for users.